py3status/modules/apt_updates.py

```diff
@@ -28,13 +28,18 @@
 import sys
 
 LINE_SEPARATOR = "\\n" if sys.version_info > (3, 0) else "\n"
+STRING_NOT_INSTALLED = "isn't installed"
```

Comment:
`not installed' might be easier to read

Reply:
Ya. I agree. Does not look decent anymore in the long run. I plan to ask you about replacing all of them too (in the PR where I added bunch of empty docstrings). For now, just use `isn't installed` and we'll change them all.

Reply:
I'd prefer to just do it once.

This is why just having a few PRs in the queue is less work for you as well as for me/@ultrabug. So I'm going to wait on them to be correct.

Reply:
K. Fixed this one.